from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_AddModel(object):
    def setupUi(self, AddModel):
        AddModel.setObjectName("AddModel")
        AddModel.resize(400, 235)
        AddModel.setMinimumSize(QtCore.QSize(400, 235))
        AddModel.setMaximumSize(QtCore.QSize(400, 234))
        self.name = QtWidgets.QLabel(AddModel)
        self.name.setGeometry(QtCore.QRect(30, 70, 60, 20))
        self.name.setObjectName("name")
        self.nameBox = QtWidgets.QLineEdit(AddModel)
        self.nameBox.setGeometry(QtCore.QRect(100, 70, 260, 20))
        self.nameBox.setObjectName("nameBox")
        self.DIR = QtWidgets.QLabel(AddModel)
        self.DIR.setGeometry(QtCore.QRect(30, 30, 60, 20))
        self.DIR.setObjectName("DIR")
        self.DIRBox = QtWidgets.QLineEdit(AddModel)
        self.DIRBox.setGeometry(QtCore.QRect(100, 30, 201, 20))
        self.DIRBox.setObjectName("DIRBox")
        self.selectBtn = QtWidgets.QPushButton(AddModel)
        self.selectBtn.setGeometry(QtCore.QRect(310, 29, 50, 22))
        self.selectBtn.setObjectName("selectBtn")
        self.comment = QtWidgets.QLabel(AddModel)
        self.comment.setGeometry(QtCore.QRect(30, 150, 60, 20))
        self.comment.setObjectName("comment")
        self.commentBox = QtWidgets.QLineEdit(AddModel)
        self.commentBox.setGeometry(QtCore.QRect(100, 150, 260, 20))
        self.commentBox.setObjectName("commentBox")
        self.cancelBtn = QtWidgets.QPushButton(AddModel)
        self.cancelBtn.setGeometry(QtCore.QRect(110, 190, 80, 25))
        self.cancelBtn.setObjectName("cancelBtn")
        self.comfirmBtn = QtWidgets.QPushButton(AddModel)
        self.comfirmBtn.setGeometry(QtCore.QRect(230, 190, 80, 25))
        self.comfirmBtn.setObjectName("comfirmBtn")
        self.saveDIR = QtWidgets.QLabel(AddModel)
        self.saveDIR.setGeometry(QtCore.QRect(30, 110, 60, 20))
        self.saveDIR.setObjectName("saveDIR")
        self.saveDIRBox = QtWidgets.QLineEdit(AddModel)
        self.saveDIRBox.setGeometry(QtCore.QRect(100, 110, 201, 20))
        self.saveDIRBox.setObjectName("saveDIRBox")
        self.saveSelectBtn = QtWidgets.QPushButton(AddModel)
        self.saveSelectBtn.setGeometry(QtCore.QRect(310, 109, 50, 22))
        self.saveSelectBtn.setObjectName("saveSelectBtn")

        self.retranslateUi(AddModel)
        QtCore.QMetaObject.connectSlotsByName(AddModel)

    def retranslateUi(self, AddModel):
        _translate = QtCore.QCoreApplication.translate
        AddModel.setWindowTitle(_translate("AddModel", "添加模型"))
        self.name.setText(_translate("AddModel", " 名    称"))
        self.DIR.setText(_translate("AddModel", " 文件位置"))
        self.selectBtn.setText(_translate("AddModel", "选 取"))
        self.comment.setText(_translate("AddModel", " 描    述"))
        self.cancelBtn.setText(_translate("AddModel", "取 消"))
        self.comfirmBtn.setText(_translate("AddModel", "确 认"))
        self.saveDIR.setText(_translate("AddModel", " 保存位置"))
        self.saveSelectBtn.setText(_translate("AddModel", "选 取"))
